auth-web/src/views/auth/home/IncorpOrRegisterView.vue

```diff
@@ -66,10 +94,29 @@ import { appendAccountId } from 'sbc-common-components/src/util/common-util'
 export default class IncorpOrRegisterView extends Vue {
   protected readonly learnMoreUrl = 'https://www2.gov.bc.ca/gov/content/governments/organizational-structure/ministries-organizations/ministries/citizens-services/bc-registries-online-services'
   private readonly bulletPoints: Array<any> = [
-    { text: 'For Named Companies, add your existing Name Request number to your account and open it.' },
-    { text: 'Establish your company\'s articles and prepare an Incorporation Agreement. Either create your own, or use template provided in the Incorporation Application.' },
-    { text: 'Complete the Incorporation Application by providing information about your company: addresses, directors and share structure.' },
-    { text: 'Retain a copy of all Incorporation documents for your business\'s records.' }
+    { text: 'Register a firm such as sole proprietorship, DBA, general partnership, and societies' },
+    { text: 'Incorporate a benefit company or a cooperative association' }
+  ]
+
+  private readonly expansionPanels: Array<any> = [
+    { text: 'Sole Proprietorship, DBA, and General Partnership',
```

Comment:
en.json is another good place to keep text

Reply:
Hmm. It doesn't look like a good place for this array style text. I'll keep it in mind though.